pulp_ansible/tests/functional/api/collection/test_sync.py

```diff
@@ -184,9 +184,7 @@ def test_mirror_sync_with_requirements(self):
         # Step 4
         sync(self.cfg, collection_remote, repo, mirror=True)
         repo = self.client.get(repo["_href"])
-        expected = ANSIBLE_COLLECTION_FIXTURE_SUMMARY
-        expected["ansible.collection_version"] = 2
-        self.assertDictEqual(get_added_content_summary(repo), expected)
+        self.assertDictEqual(get_added_content_summary(repo), ANSIBLE_COLLECTION_FIXTURE_SUMMARY)
```

Comment:
What about we update the usage of  `assertDictEqual`? We could use another assertion a bit more strict. Removing the need of `mock.ANY`, for instance `assertGreaterEqual(a, b)`.  Since the end goal is to assure that content was synced properly, but we do not control that URL being synced.

Reply:
sounds good to me